### DISCUSSION TITLE
组件中t-class class的区别

### DISCUSSION THREAD

**Discussion by maclxf** [Tencent/tdesign-miniprogram#3189] [Q&A]
![image](https://github.com/user-attachments/assets/746f6e25-1a9b-4a2b-a6b7-ce437417b9c1)
t-class是根节点样式类，但是这个class是什么呢？

**Comment by maclxf** [comment]
找到个相关的PR，但是没找到解释
https://github.com/Tencent/tdesign-miniprogram/pull/1587

**Comment by jarmywang** [comment]
@maclxf [官方文档](https://developers.weixin.qq.com/miniprogram/dev/framework/custom-component/wxml-wxss.html#%E5%A4%96%E9%83%A8%E6%A0%B7%E5%BC%8F%E7%B1%BB)有一句话：
> 在同一个节点上使用普通样式类和外部样式类时，两个类的优先级是未定义的，因此最好避免这种情况。

我猜测应该是加一个叫class的外部样式类让用户使用class与t-class都视为外部样式，降低这类风险，未验证

**Comment by anlyyao** [comment]
是因为 virtualHost ～

see: https://developers.weixin.qq.com/miniprogram/dev/framework/custom-component/wxml-wxss.html#%E8%99%9A%E6%8B%9F%E5%8C%96%E7%BB%84%E4%BB%B6%E8%8A%82%E7%82%B9

**Reply by maclxf** [reply]
诶，不是特别能理解这两者的关系，望明示😥
